frontend/src/forms/subforms/PidPinForm.tsx

```diff
@@ -33,6 +34,17 @@ const PidPinForm: FunctionComponent<PidPinProps> = (props: PidPinProps) => {
     const { nameSpace } = props;
     return nameSpace ? `${nameSpace}.${fieldName}` : fieldName;
   };
+
+  const keycloak = useKeycloakWrapper();
+
+  const disableProjectNumber = () => {
+    if (keycloak.hasClaim('admin-properties')) {
```

Comment:
I would change this constant to projectNumberDisabled and
`const projectNumberDisabled = !keycloak.hasClaim(Claims.ADMIN_PROPOERTIES)`

Reply:
Ooh that is way cleaner, on it.